osu.Game/Screens/OnlinePlay/Components/OnlinePlayBackgroundScreen.cs

```diff
@@ -59,9 +60,10 @@ private void updateBackground()
         {
             Schedule(() =>
             {
-                var beatmap = playlistItem?.Beatmap.Value;
+                string? lastCover = (background?.Beatmap?.BeatmapSet as IBeatmapSetOnlineInfo)?.Covers.Cover;
+                string? newCover = (background?.Beatmap?.BeatmapSet as IBeatmapSetOnlineInfo)?.Covers.Cover;
```

Comment:
This one should be coming from the new beatmap (`beatmap?.BeatmapSet` I believe, judging by the previous code).

Reply:
Oops, fixed but pushed to the wrong branch.